Freshpaint/Classes/FPAnalytics.m

```diff
@@ -567,4 +570,13 @@ - (void)run:(FPEventType)eventType payload:(FPPayload *)payload
     [self.runner run:context callback:nil];
 }
 
+- (NSString *)validatedSessionId
+{
+    NSTimeInterval timeout = self.state.configuration.sessionTimeout ?: 1800;
```

Comment:
instead of defaulting to `1800` in the method body here, can we default `self.state.configuration.sessionTimeout` itself to `1800` if no custom timeout has been set? 

i think this would be more declarative and make it easier to find out what the default timeout is if someone was looking.

Reply:
nvm it looks like we already do this, but this is like an extra extra fallback to `1800`. maybe we can have the value as a constnt instead of hard-coding it.